configurations/icl_machakos_debate_2022.py

```diff
@@ -102,7 +102,7 @@
                         CodeSchemeConfiguration(code_scheme=load_code_scheme("rqas/icl/machakos_county_priorities_2022"),
                                                 auto_coder=None, coda_code_schemes_count=3),
                     ],
-                    ws_code_match_value="icl_machakos_county_priorities_2022"
+                    ws_code_match_value="machakos_county_priorities_2022"
```

Comment:
If this is in project icl_machakos then shouldn't we call this icl_machakos_...?